src/main/java/org/folio/dataexp/service/export/strategies/ld/AbstractLinkedDataExportStrategy.java

```diff
@@ -0,0 +1,245 @@
+  protected LocalStorageWriter createLocalStorageWriter(
+      JobExecutionExportFilesEntity exportFilesEntity
+  ) {
+    return new LocalStorageWriter(
+        S3FilePathUtils.getLocalStorageWriterPath(
+            exportTmpStorage,
+            exportFilesEntity.getFileLocation()
+        ),
+        OUTPUT_BUFFER_SIZE
+    );
+  }
+
+  /**
+   * Process the whole set of export IDs in slices, where each slice is turned into a
+   * set and the real work of retrieving, converting, and writing is done for each set.
+   * Note that the analogous MARC-generating processing tracks duplication errors, but
+   * due to the way the only Linked Data implementation (so far) works, the input will
+   * always be a set of UUIDs, and the retrieval mechanism should never respond with
+   * more than one matching resource, so duplicate errors aren't tracked here.
+   *
+   * @param exportFilesEntity export job
+   * @param exportStatistic export job statistics collector
+   * @param localStorageWriter output writer
+   */
+  protected void processSlices(
+      JobExecutionExportFilesEntity exportFilesEntity,
+      ExportStrategyStatistic exportStatistic,
+      LocalStorageWriter localStorageWriter
+  ) {
+    var slice = exportIdEntityRepository.getExportIds(
+        exportFilesEntity.getJobExecutionId(),
+        exportFilesEntity.getFromId(),
+        exportFilesEntity.getToId(),
+        PageRequest.of(0, exportIdsBatch)
+    );
+    log.info("Slice size: {}", slice.getSize());
+    var exportIds = slice.getContent().stream()
+        .map(ExportIdEntity::getInstanceId)
+        .collect(Collectors.toSet());
+    createAndSaveLinkedData(
+        exportIds,
+        exportStatistic,
+        exportFilesEntity.getJobExecutionId(),
+        localStorageWriter
+    );
+    while (slice.hasNext()) {
+      slice = exportIdEntityRepository.getExportIds(
+          exportFilesEntity.getJobExecutionId(),
+          exportFilesEntity.getFromId(),
+          exportFilesEntity.getToId(),
+          slice.nextPageable()
+      );
+      exportIds = slice.getContent().stream()
+          .map(ExportIdEntity::getInstanceId)
+          .collect(Collectors.toSet());
+      createAndSaveLinkedData(
+          exportIds,
+          exportStatistic,
+          exportFilesEntity.getJobExecutionId(),
+          localStorageWriter
+      );
+    }
+  }
+
+  /**
+   * Retrieve, convert, and write Linked Data resources.
+   *
+   * @param externalIds set of instance identifiers to retrieve
+   * @param exportStatistic export job statistics collector
+   * @param jobExecutionId export job identifier
+   * @param localStorageWriter output writer
+   */
+  protected void createAndSaveLinkedData(
+      Set<UUID> externalIds,
+      ExportStrategyStatistic exportStatistic,
+      UUID jobExecutionId,
+      LocalStorageWriter localStorageWriter
+  ) {
+    var resources = getLinkedDataResources(externalIds);
+    log.info("linkedDataResources size: {}", resources.size());
```

Comment:
Will this be useful for linked data export troubleshooting? If no - then maybe you can remove it or make DEBUG level.